setup.py

```diff
@@ -3,7 +3,42 @@
 
 """This package includes dependencies of the openfl project."""
 
+from subprocess import check_call
+from sys import executable
+
 from setuptools import setup
+from setuptools.command.build_py import build_py
+
+
+class GRPCBuildPyCommand(build_py):
+    """Command to generate project *_pb2.py modules from proto files."""
+
+    def run(self):
+        """Build Python and GRPC modules."""
+        check_call([executable, '-m', 'pip', 'install', 'grpcio-tools~=1.34.0'])
```

Comment:
are protobuf files built before we install requirements? 
In other case I do not see why we call `pip` here explicitly

Reply:
Yes, on this step we do not have any of the requirements installed.